css/css-properties-values-api/META.yml

```diff
@@ -872,13 +873,29 @@ links:
           subtest: CSSOM setters function as expected for unregistered properties
         - test: registered-property-cssom.html
           subtest: Initially unregistered values from stylesheet
+        - test: at-property-cssom.html
+          subtest: Rule for --valid returns expected value for CSSPropertyRule.initialValue
+        - test: at-property-cssom.html
+          subtest: Rule for --valid-reverse returns expected value for CSSPropertyRule.initialValue
+        - test: at-property-cssom.html
+          subtest: Rule for --valid-whitespace returns expected value for CSSPropertyRule.initialValue
+        - test: at-property-cssom.html
+          subtest: Rule for --vALId returns expected value for CSSPropertyRule.initialValue
+        - test: at-property-cssom.html
+          subtest: Rule for --no-syntax returns expected value for CSSPropertyRule.initialValue
+        - test: at-property-cssom.html
+          subtest: Rule for --no-inherits returns expected value for CSSPropertyRule.initialValue
+        - test: at-property-cssom.html
+          subtest: Rule for --initial-value-only returns expected value for CSSPropertyRule.initialValue
     - product: chrome
       url: https://crbug.com/981537
       results:
         - test: register-property-syntax-parsing.html
           subtest: syntax:'<image>', initialValue:'url(a)' is valid
         - test: register-property-syntax-parsing.html
           subtest: syntax:'<image>', initialValue:'linear-gradient(yellow, blue)' is valid
+        - test: register-property-syntax-parsing.html
+          subtest: syntax:'<color> | <image> | <url> | <integer> | <angle>', initialValue:'red' is valid
```

Comment:
Need this

Reply:
Update:
This is now covered by https://github.com/web-platform-tests/wpt-metadata/pull/3467